scripts/ziti-builder.sh

```diff
@@ -62,7 +62,7 @@ function set_workspace(){
     else
         echo -e "INFO: project not mounted on ${WORKSPACE}"\
                 "\nINFO: mounting on ziti-builder container"
-        docker pull "openziti/ziti-builder:${ZITI_BUILDER_TAG:-latest}"
+        docker pull "openziti/ziti-builder:${ZITI_BUILDER_TAG:-latest}" &>/dev/null || true
```

Comment:
Why is it ok for the pull to fail?

Reply:
It's a local cross-build script that supports development of ZET binary and packages and emulates the CI environment. It may also be useful for producing one-offs for a proof of concept.

The pull will fail if the selected builder image exists locally but not in the registry, and the script will raise the exception if the `docker run` command, too, fails to match an image.